server/server.js

```diff
@@ -15,22 +18,30 @@ passport.deserializeUser((id, done) => {
   done(null, id);
 });
 
-// passport-local configurtion
+// passport-local configuration
 passport.use(new LocalStrategy({
     usernameField: 'email'
   },
   function(email, password, done) {
-    User.findOne({where: {email}}, (err, user) => {
-      if (err) {
-        return done(err);
-      }
-
-      if (!user || !user.verifyPassword(password)) {
-        return done(null, false);
-      }
-
-      return done(null, user);
-    });
+    User.findOne({where: {email}})
+        .then((user) => {
+          if (!user) {
+            return done(null, false);
+          }
+
+          // compare password to hashed password from db
+          bcrypt.compare(password, user.password, (err, correctPassword) => {
```

Comment:
Can we create an instance method on User which uses bcrypt to check the password? (We shouldn't be requiring bcrypt in server.js)